static/app/components/performance/spanSearchQueryBuilder.tsx

```diff
@@ -104,6 +157,7 @@ export function SpanSearchQueryBuilder({
       placeholder={placeholderText}
       filterKeys={filterTags}
       initialQuery={initialQuery}
+      fieldDefinitionGetter={(key: string) => getFieldDefinition(key, 'span')}
```

Comment:
can you extract this out to its own function so the reference stays the same on rerenders?

Reply:
Done